tests/lib/virt.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import libvirt
+import time
+
+import xml.etree.ElementTree as ET
+
+from lib.log import log
+
+# filled in by connectDomain function
+snapshots = None
```

Comment:
avoid global state. I recommend wrapping this in a class. The functions in this package can be methods.